schema/common/defaults/configuration/kubernetes-node.yml

```diff
@@ -6,3 +6,4 @@ specification:
   version: 1.22.4
   cni_version: 0.8.7
   node_labels: "node-type=epiphany"
+  enable_controller_attach_detach: false
```

Comment:
As proposed by @to-bar, we can move this argument to kubernetes-master configuration.
Name of the file we want to move it to: `schema/common/defaults/configuration/kubernetes-master.yml`